### PR TITLE
lubridate call fixes

### PR DESCRIPTION
**Pull request by mdietze**
bunch of lubridate functions that now need a lubridate:: prefix.

In general, as we switch packages over we need to (A) be careful not to miss functions and (B) be vigilant that we will get this sorts of errors and will need to test the next release extensively

### CODE REVIEW

**Comment by bpbond** [comment]
@mdietze were these generating errors?

**Comment by mdietze** [comment]
Yes, the workflow was stopping when it hit these. Roughly a 1:1 mapping between commits and attempted runs

**Comment by bpbond** [comment]
Huh. Ok.

File: modules/data.atmosphere/R/met2CF.ALMA.R [hunk 1]
```diff
@@ -102,7 +102,7 @@ met2CF.PalEON <- function(in.path, in.prefix, outfolder, start_date, end_date, l
           met[[v]] <- ncvar_get(nc = nc1, varid = v)
         } else {
           tmp      <- ncvar_get(nc = nc1, varid = v)
-          met[[v]] <- abind(met[[v]], tmp)
+          met[[v]] <- abind::abind(met[[v]], tmp)
```
Comment (serbinsh) [inline modules/data.atmosphere/R/met2CF.ALMA.R]: @mdietze just checking, just importing abind from abind function to use here correct?  That is 

"Combine multi-dimensional arrays. This is a generalization of cbind and rbind. Takes a sequence
of vectors, matrices, or arrays and produces a single array of the same or higher dimension."

Comment (mdietze) [inline modules/data.atmosphere/R/met2CF.ALMA.R]: yep. variables have time, lat, lon dimensions so can't just rbind

File: db/R/dbfiles.R [hunk 2]
```diff
@@ -59,10 +59,10 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
   inputid <- NULL
   if (nrow(existing.input) > 0) {
     # Convert dates to Date objects and strip all time zones (DB values are timezone-free)
-    startdate <- force_tz(as_date(startdate), 'GMT')
-    enddate <- force_tz(as_date(enddate), 'GMT')
-    existing.input$start_date <- force_tz(as_date(existing.input$start_date), 'GMT')
-    existing.input$end_date <- force_tz(as_date(existing.input$end_date), 'GMT')
+    startdate <- lubridate::force_tz(lubridate::as_date(startdate), 'GMT')
```
Comment (serbinsh) [inline db/R/dbfiles.R]: @mdietze Should these be forced to UTC instead of GMT?

**Comment by ashiklom** [comment]
@bpbond `Imports` loads the packages, but doesn't add them to the search path, which is where R looks for packages. It's very confusing. More details [here](http://r-pkgs.had.co.nz/namespace.html#search-path).

**Pull request merged**